src/main/java/org/raystack/depot/config/MetricsConfig.java

```diff
@@ -15,4 +15,8 @@ public interface MetricsConfig extends Config {
     @Config.Key("METRIC_STATSD_TAGS")
     @DefaultValue("")
     String getMetricStatsDTags();
+
+    @Config.Key("METRIC_STATSD_TAGS_NATIVE_FORMAT_ENABLE")
+    @DefaultValue("")
```

Comment:
empty string as default value for boolean config?

Reply:
fixed